src/components/calcite-inline-editable/calcite-inline-editable.scss

```diff
@@ -1,3 +1,7 @@
+:host {
+  @apply inline-block;
```

Comment:
@eriklharper what display should this component have? Should it be "block" like input?